Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Screens/Screen.cs

```diff
@@ -9,7 +9,8 @@
 
 namespace Aquality.Appium.Mobile.Screens
 {
-    public abstract class Screen<T> where T : AppiumDriver<AppiumWebElement>
+    public abstract class Screen<T> : IScreen
+        where T : AppiumDriver<AppiumWebElement>
```

Comment:
please implement IParent interface to ease the parent element creation

Reply:
let's do it in serate task, it does not relate to screenfactory